compiler/luci/pass/src/CircleOptimizer.cpp

```diff
@@ -26,6 +26,7 @@
 #include "luci/Pass/FoldGatherPass.h"
 #include "luci/Pass/FoldSparseToDensePass.h"
 #include "luci/Pass/ForwardReshapeToUnaryOpPass.h"
+#include "luci/Pass/ForwardTransposeOpPass.h"
```

Comment:
I think that it is not necessary to split the pass according to arity.

If we split the pass, users will have to turn on multiple options that basically do the same thing (forwarding Transpose).

ForwardTransposeToUnary..
ForwardTransposeToBinary..
ForwardTransposeToMultiInput...

We can split the passes internally, but I think it would be better to expose a single option to users.